### PR TITLE
Diagram-name: Correct the name of the created diagram file

### PR DESCRIPTION
**Pull request by PerninGuillaume**
#35  
Signed-off-by: Guillaume Pernin <guillaume.pernin@rte-france.com>

### CODE REVIEW

**Review by rosiereflo — CHANGES_REQUESTED** [review]
Could you please remove the lines in tests/main/CMakeList.txt that wrongly indicate the the test with diagram should fail?

**Review by marcochiaramello — APPROVED** [review]

File: sources/Context.cpp [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include "Message.hpp"
 #include "Par.h"
 #include "Diagram.h"
+#include "Outputs/src/Constants.h"
```
Comment (lecourtoisflo) [inline sources/Context.cpp:27]: replace by "Constant.h" and put Constant.h in includes of library output

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]
comment for description of diagramFileSuffixExt is incorrect

**Comment by PerninGuillaume** [comment]
The launch test now fails, in order to pass it will need the last modifications Marco brought in its lasts PR

**Review by rosiereflo — APPROVED** [review]

**Review by lecourtoisflo — APPROVED** [review]

**Pull request merged**